cabal-testsuite/PackageTests/Regression/T4025/setup.test.hs

```diff
@@ -3,7 +3,10 @@ import Test.Cabal.Prelude
 -- an executable RPATH.  Don't test on Windows, which doesn't
 -- support RPATH.
 main = setupAndCabalTest $ do
-    skipIfWindows
+  skipIfWindows
```

Comment:
maybe we could move this skipIfWindows to ` expectBrokenIf ((osx || win) && ghc)`

maybe it should be in other pr, revising all skipIfWindows and transform as much of them as possible in knownBrokenIfWindows

Reply:
Yeah, I think I'd prefer to leave this out. Skipping might also be more appropriate (in some of those `ifWindows` cases at least), where we test non-Windows functionality, instead of something that we'd usually expect to work but that's broken for some reason.

(There's a second difference -- `expectBrokenIf` tests will be run, while `skipIf` won't. So e.g. for tests that are too slow, we can't use `expectBrokenIf`.)